demos/text_detection_demo/cpp/models.lst

```diff
@@ -1,4 +1,4 @@
 # This file can be used with the --list option of the model downloader.
 handwritten-score-recognition-????
 text-detection-????
-text-recognition-????
+text-recognition-*
```

Comment:
I think it may be useful to stay some hints regarding different models usage in model.lst

Reply:
Changed normalization, change ac metrics and add info about normalization to README.md. Add information about `decoder_type` parameter to models.lst.